src/logger.js

```diff
@@ -1,38 +1,121 @@
+const dotenv = require('dotenv');
+
 /* istanbul ignore file */
+const { LOGLEVELS, structuredLogger } = require('@rudderstack/integrations-lib');
 
-const levelDebug = 0; // Most verbose logging level
-const levelInfo = 1; // Logs about state of the application
-const levelWarn = 2; // Logs about warnings which dont immediately halt the application
-const levelError = 3; // Logs about errors which dont immediately halt the application
 // any value greater than levelError will work as levelNone
+const loggerImpl = process.env.LOGGER_IMPL ?? 'winston';
```

Comment:
Is LOGGER_IMPL new env variable getting introduced ?
Since we want to go default with winston, are we good with not mentioning this variable anywhere in helm charts ?

Reply:
I guess it should be ok.

Reply:
Added comments for more clarity